sdk/opa.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2021 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package sdk
+
+import (
+	"context"
+	"crypto/rand"
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	"github.com/pkg/errors"
```

Comment:
[nit] There's little reason for this package in go nowadays, I think:

```go
errors.Wrap(err, "foo went bad")
```
is roughly equivalent to
```go
fmt.Errorf("foo went bad: %w", err)
```

where `%w` is a special directive that wraps the error, and let's callers unwrap it.

Reply:
Ah, thanks! 👍